charts/fluentd/templates/configmap-extra-config.yaml

```diff
@@ -0,0 +1,21 @@
+{{- if or (.Values.output.extraConfigs) ( and (.Values.output.plugins.enabled) (gt (len .Values.output.plugins.pluginsList) 0)) -}}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ template "fluentd.fullname" . }}-additionalconfigs
+  labels:
+    {{- include "fluentd.labels" . | nindent 4 }}
+data:
+{{- range $key, $value := .Values.configMaps }}
+  {{ $key }}: |-
+{{ $value | indent 4 }}
+{{- end }}
```

Comment:
what does this do?

Reply:
This iterates over .Values.configMaps key:value

so each fluentd.config: <YAML> other.config: >YAML> will be mapped the same way to the configMap data.